mapswipe_workers/mapswipe_workers/utils/slack_helper.py

```diff
@@ -58,6 +58,24 @@ def send_slack_message(
             + "and activate another one."
         )
         slack_client.chat_postMessage(channel="mapswipe_managers", text=message)
+    elif message_type == MessageType.PROJECT_STATUS_FINISHED:
+        message = (
+            "### SET PROJECT STATUS TO FINISHED ###\n"
+            + f"Project Name: {project_name}\n"
+            + f"Project Id: {project_id}\n\n"
+            + "The status of the project has been set to 'finished' "
+            + "by MapSwipe's backend workers."
+        )
+        slack_client.chat_postMessage(channel=SLACK_CHANNEL, text=message)
```

Comment:
yes there might be a better way here. The main thing is that we use two different slack channels. there is a `backend-notifications` channel which will be used when you set up a project successfully and there is the `mapswipe_managers` channel which will be used to report on the progress of projects.

for this PR I would leave things as they are because adding new env variables etc. might need more testing...